Source/Extensions/Blazorise.DataGrid/DataGrid.razor.cs

```diff
@@ -2097,6 +2097,23 @@ public IEnumerable<TItem> DisplayData
         [Parameter( CaptureUnmatchedValues = true )]
         public Dictionary<string, object> Attributes { get; set; }
 
+        /// <summary>
+        /// Returns a zero-based index of the currently selected row if found; otherwise it'll return -1.
+        /// Considers the current pagination.
+        /// </summary>
+        public int SelectedRowIndex

```

Comment:
I wonder if `SelectedRowIndex` needs to have two-way binding? What if the user would like to do `@bind-SelectedRowIndex="@selectedRowIndex"`, instead of `@bind-SelectedRow="@selectedRow"`?

Reply:
Hahaha, I know, I can't help it. I remember when I used `SelectedRowIndex` in the past when I worked with another UI library. It was really helpful. But since it is a parameter, even if we add a two-way binding in the future it will not be a breaking change. So it is good for now as it is.